### DISCUSSION TITLE
What abilities should flocktraces have?

### DISCUSSION THREAD

**Discussion by amylizzle** [stonepillars/goon-flock#13] [Ideas]
Flocktraces can currently put down blueprints, which can basically only result in blueprint spam which cripples the flock. I think we're all fairly agreed that traces shouldn't be able to blueprint, but what _should_ they be able to do?

Current flockmind abilities:

- Designate priority tile
- Designate enemy
- Partition mind
- Diffract drone
- Concentrated repair burst
- Gatecrash
- Radio stun burst
- Narrowbeam transmission
- Fabricate structure

Current flocktrace abilities:
- Fabricate Structure

**Comment by stonepillars** [comment]
> * Designate enemy

This is the only one that seems to be out of the Flockmind's lane, and the Flocktrace probably should be for controlling a drone so that they are more smart / coordinate better.

**Comment by amylizzle** [comment]
Maybe some new ones?
Like:
rally-all-on-screen. That way you've got flocktraces acting like sergeants for their squad of drones.
item-dump - cause manufacturers and vending machines to spit out items. 
a reduced version of gate crash?

**Reply by stonepillars** [reply]
Maybe flock traces could be tied to some sort of building, and that's where they live. They can directly command drones they create using the abilities above, while the parent flockmind has control over them all.

**Reply by TobleroneSwordfish** [reply]
rally-all-on-screen is definitely something the flockmind should have
I like ways of generating more items since that's the flock's primary means of resources generation (would be nice to have more ways of generating resources in future)
